src/js/background/backgroundLogic.js

```diff
@@ -113,11 +113,12 @@ const backgroundLogic = {
     return list.concat(containerState.hiddenTabs);
   },
 
-  async unhideContainer(cookieStoreId) {
+  async unhideContainer(cookieStoreId, toHide) {
```

Comment:
Can we change this variable name from `toHide` to `alreadyShowingUrl` to make it more explicit?